api/src/main/java/uk/gov/dhsc/htbhf/dwp/model/EligibilityResponse.java

```diff
@@ -18,6 +18,10 @@
     @ApiModelProperty(notes = "The eligibility status", example = "ELIGIBLE")
     private EligibilityStatus eligibilityStatus;
 
+    @JsonProperty("earningsThresholdExceeded")
+    @ApiModelProperty(notes = "Whether or not the household has exceeded the earnings threshold for help to buy healthy foods.", example = "true")
+    private Boolean earningsThresholdExceeded;
```

Comment:
We need to go through this on a whiteboard - we may not need this flag. 
I think if we say a claimant is eligible if: 1) They are on the feed (in the db) - therefore in receipt of a qualifying benefit; 2) under the income threshold (if on UC) - then I believe this is enough information to work with. 
We should also include details of the children in the response and allow the client to decide if they're actually eligible (i.e. eligibilityStatus is ELIGIBLE and either they're pregnant or they have children under 4).

Reply:
This still needs to be removed